executor/join.go

```diff
@@ -729,6 +752,7 @@ func (e *HashJoinExec) join2Chunk(workerID int, outerChk *chunk.Chunk, joinResul
 			joinResult.err = errors.Trace(err)
 			return false, joinResult
 		}
+		e.updateMemoryUsage(workerID+1, joinResultChkBuffer.MemoryUsage())
```

Comment:
Can we only calculate the innerResult for now?
The join result is much smaller compared to innerResult.